conf/config.inc.php

```diff
@@ -288,3 +293,8 @@
 
 # Launch a posthook script after successful password change
 #$posthook = "/usr/share/self-service-password/posthook.sh";
+
+# Allow to override current settings with local configuration
+if (file_exists (dirname (__FILE__) . '/config.inc.local.php')) {
+    include dirname (__FILE__) . '/config.inc.local.php';
```

Comment:
I would have preferred a _require_ instead of _include_, _require_ will stop the execution with an error (instead of a warning with continued execution) if the config.inc.local.php is not readable (or less likely, is a directory) which is potentially a future common issue if the file is not shipped by the package.